CI/appveyor.set-build-info.ps1

```diff
@@ -1,7 +1,14 @@
 cd $Env:APPVEYOR_BUILD_FOLDER
 
 if ($Env:APPVEYOR_REPO_TAG -eq "false") {
-  $Env:MUDLET_VERSION_BUILD = "-testing"
+  # The only scheduled Appveyor builds are public test builds
+  if ($Env:APPVEYOR_SCHEDULED_BUILD -ne "false") {
+    $Env:MUDLET_VERSION_BUILD = "-public-test-build"
+  } else {
+    $Env:MUDLET_VERSION_BUILD = "-testing"
+  }
+  # hardcoded for testing
+  $Env:MUDLET_VERSION_BUILD = "-public-test-build"
```

Comment:
If we keep this as it is, all Windows builds from PRs will break because they'll try to build a ptb and fail due to the failing squirrel command.
So we should probably put a guard around here to prevent all windows builds from breaking. Maybe a check if the signing password is available?

Reply:
Good point, I've removed it, so we'll test via scheduled builds instead. Simpler